### PR TITLE
Fixing feedback

### PR DESCRIPTION
**Pull request by evelinec**
- Addressed the new comment in #40

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -104,8 +104,10 @@ The [hotspot=43]`@ArquillianResource` annotation is used to retrieve the
 `\http://localhost:9080/arquillian-managed/` base URL for this web service.
 The annotation provides the host name, port number and web archive
 information for this service, so you don't need to hardcode these values
-in the test case. Notice the `arquillian-managed` WAR name that was specified
-in the web archive, it's needed  when the `inventory` service communicates
+in the test case. The `arquillian-managed` WAR name comes
+from the WAR name you specified when you created the web archive in the
+[hotspot=36-41]`@Deployment` method.
+It's needed  when the `inventory` service communicates
```
Comment (yeekangc) [inline README.adoc]: Extra space between 'needed' and 'when'?  A nit.

File: README.adoc [hunk 2]
```diff
@@ -104,8 +104,10 @@ The [hotspot=43]`@ArquillianResource` annotation is used to retrieve the
 `\http://localhost:9080/arquillian-managed/` base URL for this web service.
 The annotation provides the host name, port number and web archive
 information for this service, so you don't need to hardcode these values
-in the test case. Notice the `arquillian-managed` WAR name that was specified
-in the web archive, it's needed  when the `inventory` service communicates
+in the test case. The `arquillian-managed` WAR name comes
```
Comment (yeekangc) [inline README.adoc]: Are we talking about the `arquillian-managed` in the URL?  If so, we can refer to that directly?  And, indicate that that part of the URL is from the WAR name defined for the web archive?

File: README.adoc (hunk 1 above)
Comment (evelinec) [reply README.adoc]: Fixed

File: README.adoc (hunk 2 above)
Comment (evelinec) [reply README.adoc]: Fixed

File: README.adoc [hunk 3]
```diff
@@ -104,8 +104,10 @@ The [hotspot=43]`@ArquillianResource` annotation is used to retrieve the
 `\http://localhost:9080/arquillian-managed/` base URL for this web service.
 The annotation provides the host name, port number and web archive
 information for this service, so you don't need to hardcode these values
-in the test case. Notice the `arquillian-managed` WAR name that was specified
-in the web archive, it's needed  when the `inventory` service communicates
+in the test case. The `arquillian-managed` WAR name in the URL comes
```
Comment (yeekangc) [inline README.adoc]:
```suggestion
in the test case. The `arquillian-managed` path in the URL comes
```

**Comment by evelinec** [comment]
Merged to proceed with ID review.

**Pull request merged**